crates/rpc/src/cairo/ext_py.rs

```diff
@@ -191,6 +126,114 @@ impl Handle {
             Err(_closed) => Err(CallFailure::Shutdown),
         }
     }
+
+    pub async fn simulate_transaction(
+        &self,
+        at_block: BlockHashNumberOrLatest,
+        gas_price: GasPriceSource,
+        diffs: Option<Arc<PendingStateUpdate>>,
+        block_timestamp: Option<StarknetBlockTimestamp>,
+        transactions: &[BroadcastedTransaction],
+        (_skip_execute, _skip_validate): (bool, bool),
```

Comment:
I don't really recall - what is the plan for supporting these flags? Does the current cairo impl support them?

Reply:
Good catch, `skip_execute` is actually unused.